website/developers/how-to-contribute.md

```diff
@@ -6,7 +6,8 @@ children:
 - { section: Create an Issue in Jira, title: Jira }
 - { section: Contributing using GitHub, title: GitHub }
 - { section: Reviews }
-- { section: Handy Places }
+- { link: code/, title: Get the Code }
+- { link: links.html, title: Handy Places }
```

Comment:
These two links are now duplicated in the menu - they can be removed from here.
![screenshot 2016-02-09 13 07 36](https://cloud.githubusercontent.com/assets/391746/12916949/2505e880-cf2e-11e5-9626-2193e96be417.png)